subprojects/gradle/impact-shared/src/main/java/com/avito/impact/util/AndroidManifestPackageParser.kt

```diff
@@ -0,0 +1,13 @@
+package com.avito.impact.util
+
+import java.io.File
+
+object AndroidManifestPackageParser {
+
+    private val manifestPackagePattern = Regex(" package=\"([a-zA-Z0-9._]+)\"")
```

Comment:
```suggestion
    private val manifestPackagePattern = Regex(" package=\"([^"]+)\"")
```

I see no reasons to know about package valid value in this case.

Reply:
agreed

Reply:
Could we use Xml Parser?

Reply:
> Could we use Xml Parser?

yes, but we don't need deep file structure knowledge. i can optimize it reading line by line